File: public/phase-2-20160620022203/C/best-friends.c
// Name : Best Friends
#include <stdio.h>
int main() {
	// input:
	// n : the number of Dima's friends
	// fingers :  the number of fingers Dima's friends will show during the counting game
	int n, fingers[100], answer=3, index;
	scanf("%d", &n);
	for(index = 0; index< n; index++) {
		scanf("%d", &fingers[index]);
	}


	// write your code here
	// store your results in `answer`

	// output
	printf("%d", answer);
	return 0;
}

File: public/phase-2-20160620022203/JAVA/best-friends.java
// Name : Best Friends
import java.util.*;
import java.lang.*;
import java.io.*;
class Main {
	public static void main (String[] args) throws java.lang.Exception {
		// input:
		// n : the number of Dima's friends
		// fingers :  the number of fingers Dima's friends will show during the counting game
		int index;
		Scanner in = new Scanner(System.in);
		int n = Integer.parseInt(in.nextLine());
		int[] fingers = new int[n];
		String[] fingers_elements = (in.nextLine()).split(" ");
		for(index=0;index<n;index++)
			fingers[index] = Integer.parseInt(fingers_elements[index]);


		// write your code here
		// store your results in `answer`

		// output
		// Dummy Data
		int answer=3;

		System.out.println(answer);

	}
}